Formula/g/gnupg@2.4.rb

```diff
@@ -0,0 +1,96 @@
+class GnupgAT24 < Formula
+  desc "GNU Pretty Good Privacy (PGP) package"
+  homepage "https://gnupg.org/"
+  url "https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.7.tar.bz2"
+  sha256 "7b24706e4da7e0e3b06ca068231027401f238102c41c909631349dcc3b85eb46"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url "https://gnupg.org/ftp/gcrypt/gnupg/"
+    regex(/href=.*?gnupg[._-]v?(2\.4(?:\.\d+)+)\.t/i)
+  end
+
+  keg_only :versioned_formula
+
+  deprecate! date: "2026-06-30", because: :unsupported
```

Comment:
Can you add a link to the support policy for this?

Reply:
That suggests it should be disabled on that date instead.

Reply:
Given, you're a maintainer you could have pointed me to the policy.
Instead, you're harassing an external contributor.
Cc: @p-linnane @MikeMcQuaid

By the power of precedent set by @carlocab for `gnupg@2.2` in https://github.com/Homebrew/homebrew-core/commit/fc02e617a138675f6b133f5a4ce97eb2e0fd9c09 , there is a 6-months grace period before DISABLE.

I've added `disable!` 6 months after EOL due to the power of precedent.

@SMillerDev , your failure to comply means I will go to private taps for both life and work.
And Homebrew project maintainers will be tainted as incompetent bullies.
I'd rather skip that and just hand this PR to the competent senior members of your community.